### PR TITLE
add missing config

### PR DESCRIPTION
**Pull request by mjudeikis**
fixes: https://github.com/openshift/openshift-azure/issues/376 

adds load balancer config to cloud config.

### CODE REVIEW

**Comment by kargakis** [comment]
https://github.com/openshift/openshift-azure/issues/850

/retest

**Comment by kargakis** [comment]
https://github.com/openshift/openshift-azure/issues/863

/retest

**Comment by kargakis** [comment]
https://github.com/openshift/openshift-azure/issues/919

/retest

**Comment by kargakis** [comment]
latest flake was https://github.com/openshift/openshift-azure/issues/863

**Comment by kargakis** [comment]
It doesn't look like this PR has gotten through a single e2e run[1] and most failures seem related to persistent storage which seems to be affected by your changes. Mind having a look locally?

[1] https://openshift-gce-devel.appspot.com/pr/openshift_openshift-azure/917

**Comment by mjudeikis** [comment]
strange. checking

File: test/e2e/specs/fakerp/admin.go [hunk 1]
```diff
@@ -68,37 +68,42 @@ var _ = Describe("Openshift on Azure admin e2e tests [AzureClusterReader][Fake]"
 
 	// check prometheus-operator and related components readiness
 	It("should start cluster-monitoring-operator correctly", func() {
-		err := wait.Poll(2*time.Second, 20*time.Minute, ready.DeploymentIsReady(cli.AppsV1.Deployments("openshift-monitoring"), "cluster-monitoring-operator"))
+		err := wait.Poll(5*time.Second, 20*time.Minute, ready.DeploymentIsReady(cli.AppsV1.Deployments("openshift-monitoring"), "cluster-monitoring-operator"))
```
Comment (kargakis) [inline test/e2e/specs/fakerp/admin.go]: I am not sure what's the benefit of the interval changes.

**Comment by mjudeikis** [comment]
lets test one more time with timeout

**Comment by jim-minter** [comment]
You can't change to a standard load balancer unless all the non-master scalesets are in the same networksecuritygroup.  This is likely to be the source of many of the test failures associated with this PR.  If you have opened new flake issues specifically for test failures associated with this PR, please consider if they should be closed.

Please close this PR, I am going to include a fixed up version as part of resolving the wider update issue.

**Comment by mjudeikis** [comment]
> You can't change to a standard load balancer unless all the non-master scalesets are in the same networksecuritygroup. 

Where this one is coming from?

**Pull request closed without merging**